addons/sale_timesheet/models/product.py

```diff
@@ -74,10 +74,13 @@ def _onchange_service_fields(self):
             if record.type == 'service' and record.service_type == 'timesheet' and \
                not (record._origin.service_policy and record.service_policy == record._origin.service_policy):
                 record.uom_id = self.env.ref('uom.product_uom_hour')
+                record.uom_po_id = self.env.ref('uom.product_uom_hour')
```

Comment:
To avoid doing the same query because of `exists` calling [here](https://github.com/odoo/odoo/blob/2d08f97c0778469b409fca23f2be5f5a98ce3df8/odoo/api.py#L578), we could do something like that: 

```suggestion
                record.uom_id = record.uom_po_id = self.env.ref('uom.product_uom_hour')
```
🙂